src/main/java/StudentSystem/Model/Course.java

```diff
@@ -0,0 +1,54 @@
+package StudentSystem.Model;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * Created by Fariha Nawaz on 13,Apr,2020.
+ */
+@Entity
+@Table(name= "course")
+public class Course {
+
+    @Id
+    @Column(name = "course_id",nullable = false)
+    private long courseId;
+    @Column(name = "course_name", nullable = false)
+    private String courseName;
+    @Column(name = "course_instructor", nullable = false)
+    private String courseInstructor;
+
+    public Course(long courseId, String courseName, String courseInstructor) {
```

Comment:
Create a blank constructor too. Why is that necessary? - we'll come back to that later.